src/main/java/org/cryptomator/ui/unlock/UnlockInvalidMountPointController.java

```diff
@@ -38,13 +38,12 @@ public class UnlockInvalidMountPointController implements FxController {
 	@FXML
 	public void initialize() {
 		var e = unlockException.get();
-		String translationKey = "unlock.error.customPath.description.generic";
-		if (e instanceof MountPointNotSupportedException) {
-			translationKey = "unlock.error.customPath.description.notSupported";
-		} else if (e instanceof MountPointNotExistsException) {
-			translationKey = "unlock.error.customPath.description.notExists";
-		}
-		dialogDescription.setFormat(resourceBundle.getString(translationKey));
+		var translationKeySuffix = switch (e) {
+			case MountPointNotSupportedException x -> "notSupported";
+			case MountPointNotExistsException x -> "notExists";
+			default -> "generic";
+		};
+		dialogDescription.setFormat(resourceBundle.getString("unlock.error.customPath.description." + translationKeySuffix));
```

Comment:
Concatenating the translation key from a prefix + suffix adds an unnecessary computation step. Let's keep strings as compile time constants, whenever possible.

```suggestion
		var translationKey = switch (e) {
			case MountPointNotSupportedException x -> "unlock.error.customPath.description.notSupported";
			case MountPointNotExistsException x -> "unlock.error.customPath.description.notExists";
			default -> "unlock.error.customPath.description.generic";
		};
		dialogDescription.setFormat(resourceBundle.getString(translationKey));
```